### ISSUE TITLE
Timeline Layout

### ISSUE THREAD

**Issue by bjw49** [bjw49/citation-network-explorer#7]
It would be nice to be able to plot the citation network in a time ordered way.

**Comment by aarontaycheehsien** [comment]
I agree. Or if it is too complicated, perhaps color the nodes based on publication year.
